app/helpers/application_helper.rb

```diff
@@ -27,9 +30,8 @@ def background_image_style(img)
     "background-image: url(\"#{img}\");"
   end
 
-  def react_component(name, props)
-    tag(:div, { id: name, data: { react_props: props } }) do
-    end
+  def react_component(id:, component:, props: {}, wrapper_tag: 'div')
+    tag.send(wrapper_tag, { id: id, "class": REACT_COMPONENT_CLASS, data: { component: component, react_props: props } }) {}
```

Comment:
This mod allows us to be more explicit about `id` because we may have multiple components with the same name on the same page.  Now we can prevent duplicate Dom id's as well as a custom wrapper tag - the new `Mailer` component needed a `span`.